deps/spidershim/test/value.cc

```diff
@@ -778,6 +778,24 @@ TEST(SpiderShim, Date) {
   EXPECT_EQ(time, Date::Cast(*date.ToLocalChecked()->ToObject())->ValueOf());
 }
 
+namespace {
```

Comment:
> There are a number of tests about external strings in v8's test-api.cc, such as ScriptUsingStringResource, ScriptUsingOneByteStringResource, and ScriptMakingExternalString. Out of curiosity, have you tried adopting those tests? Massaging the tests a bit to get them to compile is fine. And if they have parts that depend on features we don't support yet, you can comment them out too.

I looked at those, but it wasn't clear at first glance how to port them. I'll take another look, however.

> (FWIW this can happen in a different PR, not going to block this on porting those tests, but I've been trying to adopt v8 tests where possible, to make sure we agree on the API semantics and whatnot.)

I totally agree! The ideal end state would be to be able to run all of V8 tests verbatim, although that's a long-term goal, for which porting the tests is a good first step.